rholang/src/test/scala/coop/rchain/rholang/interpreter/matcher/StreamTSpec.scala

```diff
@@ -80,6 +89,19 @@ class StreamTSpec extends FlatSpec with Matchers {
 
     assert(StreamT.run(combined).value() == (reference ++ reference))
   }
+
+  it must "be stacksafe and lazy for a stacksafe F when calling dropTail" in {
+    type StreamTCoeval[A] = StreamT[Coeval, A]
+
+    val reference                    = Stream(1)
+    val huge: StreamTCoeval[Int]     = StreamT.fromStream(Coeval.now(reference))
```

Comment:
Is it ironical that you named it `huge`? Otherwise I don't get it.

Reply:
:joy: that's residue after copy+paste & modify. One could also consider it freudian :thinking: Made my day :joy:

Reply:
fixup ready, will rebase before merge